src/components/showbooks.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import Book from './book';
+import Form from './form';
+
+export default function Books() {
+  function getInitialbooks() {
+    // getting stored items
+    const temp = localStorage.getItem('books');
+    const savedBooks = JSON.parse(temp);
+    return savedBooks || [
+      { id: 1, title: 'hello', author: 'noone' },
+      { id: 2, title: 'hel', author: 'one' },
```

Comment:
- [ ] What does "hel" mean? Please 🙏🏽 correct the word for professional documentation.